package com.anisha;

import java.rmi.RemoteException;

public class MainApplication {
	
	public static void main(String[] args) {
		
		int result=0;
		CalculatorProxy calculatorProxy= new CalculatorProxy();
		CalculatorProxy cal= new CalculatorProxy();
		try {
			result = calculatorProxy.mul(2, 4);
			result= cal.add(2, 10);
			System.out.println("Add of two numbers:" +result);
			System.out.println("Multiplication of two numbers:" +result);
		} catch (RemoteException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		
	}

}
